doc/rados/configuration/auth-config-ref.rst

```diff
@@ -190,27 +190,27 @@ stores daemons keyrings inside their data directory. The default keyring
 locations, and the capabilities necessary for the daemon to function, are shown
 below.
```

Comment:
I wonder if we should drop this section entirely; the caps are out of date (should be based on profiles) and the keyring locations aren't useful information

Reply:
will remove it in a follow-up PR.

Reply:
#40911